app/screens/claim/index.tsx

```diff
@@ -1,21 +1,61 @@
-﻿﻿import React from "react";
-import { Text, View } from "react-native";
+﻿﻿import React, {useState, useEffect} from "react";
```

Comment:
Not to nit pick, but...

```suggestion
﻿﻿import React, { useState, useEffect } from "react";
```

Reply:
Done